rpc/api/eth.go

```diff
@@ -166,6 +167,20 @@ func (self *ethApi) IsMining(req *shared.Request) (interface{}, error) {
 	return self.xeth.IsMining(), nil
 }
 
+func (self *ethApi) IsSyncing(req *shared.Request) (interface{}, error) {
```

Comment:
The name on the RPC side is fine (`eth_syncing`) actually. In Go land it doesn't make much sense.